composer.json

```diff
@@ -15,7 +15,7 @@
     "require-dev": {
         "codeclimate/php-test-reporter": "^0.4.4",
         "fzaninotto/faker": "^1.6",
-        "laravel/homestead": "^5.0",
+        "laravel/homestead": "^5",
```

Comment:
Homestead follows semantic versioning and we end up doing minor releases somewhat often. It's easy to fall behind if you're sticking to a major version. With this change we'll get 5.* instead of only 5.0.*